src/utilities/utilities.js

```diff
@@ -672,3 +672,80 @@ export function valid_range(range, [min, max]) {
   if (range === undefined) return [];
   return range.map((n) => (n >= min && n <= max ? n : NaN));
 }
+
+/**
+ * Met values of typical tasks.
+ * @public
+ * @memberof reference_values
+ * @docname Met typical tasks, [met]
+ * @constant
+ * @type {Object}
+ * @property {number} Sleeping - 0.7
+ * @property {number} Reclining - 0.8
+ * @property {number} Seated_Cquiet - 1.0
+ * @property {number} Reading_seated - 1.0
+ * @property {number} Writing - 1.0
+ * @property {number} Reading_seatedTyping - 1.1
+ * @property {number} Standing_relaxed - 1.2
+ * @property {number} Filing_seated - 1.2
+ * @property {number} Flying_aircraft_routine - 1.2
+ * @property {number} Filing_standing - 1.4
+ * @property {number} Driving_a_car - 1.5
+ * @property {number} Walking_about - 1.7
+ * @property {number} Cooking - 1.8
+ * @property {number} Table_sawing - 1.8
+ * @property {number} Walking_2mph_3_2kmh - 2.0
+ * @property {number} Lifting_packing - 2.1
+ * @property {number} Seated_heavy_limb_movement - 2.2
+ * @property {number} Light_machine_work - 2.2
+ * @property {number} Flying_aircraft_combat - 2.4
+ * @property {number} Walking_3mph_4_8kmh - 2.6
+ * @property {number} House_cleaning - 2.7
+ * @property {number} Driving_heavy_vehicle - 3.2
+ * @property {number} Dancing - 3.4
+ * @property {number} Calisthenics - 3.5
+ * @property {number} Walking_4mph_6_4kmh - 3.8
+ * @property {number} Tennis - 3.8
+ * @property {number} Heavy_machine_work - 4.0
+ * @property {number} Handling_100lb_45_kg_bags - 4.0
+ * @property {number} Pick_and_shovel_work - 4.4
+ * @property {number} Basketball - 6.3
+ * @property {number} Wrestling - 7.8
+ * @example
+ * from pythermalcomfort.utilities import met_typical_tasks
```

Comment:
the example is in python

Reply:
Should I delete this example?

Reply:
write in JS

Reply:
...🤣 Can't believe that I didn't realise I write the code in python